.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -15,18 +6,23 @@ assignees: ''
 
 <hr/>
 
+### 📝 Description
+<!-- Provide a brief, detailed description of the issue you observed. -->
+
+### 👣  Reproduction steps
+<!-- What step-by-step actions did you take? -->
```

Comment:
Adding two more spots to fill out will discourage bug reports.  Let's think of a different way to achieve this aspect of the goal  (e.g. on-the-fly training on how to report a good bug)

Reply:
Until just now, I didn't really realize we'd added "Reproduction steps" either, and I don't think that should exist, since it can be captured under "Expected behavior", and confusing or incomplete expectations can be kicked back to the reporter to help coach them on how to clarify it  (allows Quality to help other folks learn and improve at how they contribute bug reports)

Reply:
@mikermcneil This PR results from a conversation with the product quality team about improving ticket quality. 

For this PR, we aim to improve operational efficiency by using the template to guide better-quality bug tickets. We can set expectations and train un-written rules (like reproduction steps go in "expected behavior") but why not include a section? That way, when anyone (Fleetie or community members) creates a ticket, it's more explicit about what is expected. While we can train Fleeties on expectations, community members will require back-and-forth. 

This PR only introduces one new field: "Description". We realized that the description is currently being crammed into the bug title, which creates a large and unwieldy title. If we include a field for this, we can have shorter bug titles that are more succinct. 

The product quality team feels this change will result in a net improvement in ticket quality and efficiency. Clear sections in the template will serve as a reminder and reduce back-and-forth. 

If you disagree, we'll commit to your decision and work to train folks by sending tickets back to them when they lack an appropriate level of detail.